src/deps/zlib.win32.zig

```diff
@@ -10,7 +10,7 @@ const voidpf = *anyopaque;
 const Bytef = [*]u8;
 const Byte = u8;
 const uInt = c_uint;
-const uLong = c_ulong;
+const uLong = u64;
```

Comment:
is this right? cc @cirospaciari

Reply:
without this change zlib errors with `InvalidArgument` because internally it checks for expected size of structs
<img width="738" alt="Screenshot 2024-01-10 at 2 13 41 PM" src="https://github.com/oven-sh/bun/assets/35280289/ef7e356c-2267-48f9-8292-b3d4bd4df102">